### PR TITLE
Lks pr 1

### PR DESCRIPTION
**Pull request by linux-lukas**
Linux_default_install.sh script added.

### CODE REVIEW

File: linux_default_install.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+echo '+++ What does the script do exactly? +++'
+echo ''
+echo '[INFO] 1. First of all, it brings your system up to date.'
+echo ''
+echo '[INFO] 2. It installs various free and open source software solutions that are useful for both educational, professional, and personal use with Linux. '
+echo ''
```
Comment (pintman) [inline linux_default_install.sh:7]: Wrap long lines.

File: linux_default_install.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+echo ''
+while :
+do
+  read -p '[INFO] Do they agree to the script performing automated actions? (y/n)? ' ANSW
+  case $ANSW in
+	y)
+		echo '[INFO] You have chosen Yes (y)!'
+        echo '[INFO] The command line will now ask you once for your sudo password.'
+        sudo apt update -y && sudo apt upgrade -y && sudo apt full-upgrade -y && sudo apt dist-upgrade -y && sudo apt autoremove -y sudo apt autoclean -y
```
Comment (pintman) [inline linux_default_install.sh:19]: Don't do a full dist upgrade just to install some packages.

File: linux_default_install.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+while :
+do
+  read -p '[INFO] Do they agree to the script performing automated actions? (y/n)? ' ANSW
+  case $ANSW in
+	y)
+		echo '[INFO] You have chosen Yes (y)!'
+        echo '[INFO] The command line will now ask you once for your sudo password.'
+        sudo apt update -y && sudo apt upgrade -y && sudo apt full-upgrade -y && sudo apt dist-upgrade -y && sudo apt autoremove -y sudo apt autoclean -y 
+        sudo apt install -y vim vlc vscodium xampp dbeaver-ce wireshark obs-studio arduino veracrypt texlive-full wireguard git pdfsam bitbleach gimp mumble paperwork-gtk paperwork-gtk-l10n-fr firejail # TODO: Add FLOSS.
+        sudo apt update -y && sudo apt upgrade -y && sudo apt full-upgrade -y && sudo apt dist-upgrade -y && sudo apt autoremove -y sudo apt autoclean -y
```
Comment (pintman) [inline linux_default_install.sh:21]: line repeated.

File: linux_default_install.sh [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+echo '[INFO] 3. After all free and open-source software solutions have been successfully installed, it will once again bring the system completely up to date.'
+echo ''
+echo ''
+echo ''
+while :
+do
+  read -p '[INFO] Do they agree to the script performing automated actions? (y/n)? ' ANSW
+  case $ANSW in
+	y)
+		echo '[INFO] You have chosen Yes (y)!'
```
Comment (pintman) [inline linux_default_install.sh:17]: Indentation mismatch.

**Comment by pintman** [comment]
What is this PR doing? Add a descriptive name in the title. 
Remove empty files.

**Comment by linux-lukas** [comment]
I apologise for the irritation! The pull request should not be so unstructured. I think I need to get to grips with the practical operation of git and the programming of .sh scripts more directly!

**Comment by pintman** [comment]
No problem. Git is not easy github neither. :) Looking forward to see a more elaborate PR in the future. Take your time. Learn from one PR and open new PRs only after the first one has been included successfully.

Give shellcheck a try for improving shell scripts. https://www.shellcheck.net/

**Comment by linux-lukas** [comment]
Thanks for the suggestion! I will have a look at it, @pintman !

I can recommend [this Udemy course on git](https://www.udemy.com/course/git-komplett/).

**Pull request closed without merging**